lapis/request.moon

```diff
@@ -155,7 +155,7 @@ class Request
     @buffer = {} -- output buffer
     @params = {}
     @options = {}
-    @add_headers: (headers) =>
```

Comment:
This mistake is probably a side-effect of writing too much CSS during the days leading up to this commit